packages/headless/src/features/facets/facet-search-set/specific/specific-facet-search-set-slice.ts

```diff
@@ -112,6 +119,13 @@ export const specificFacetSearchSetReducer = createReducer(
   }
 );
 
+export function namespaceCommerceFieldSuggestionFacet(facetId: string): string {
+  if (facetId.includes(':')) {
+    return facetId;
+  }
+  return 'field_suggestion:' + facetId;
+}
```

Comment:
```suggestion
  const prefix = 'field_suggestion:'
  if (facetId.startsWith(prefix)) {
    return facetId;
  }
  return `${prefix}${facetId}`;
}
```

I believe it would also be preferable to define / export the `field_suggestion:` value as a constant somewhere since it's used in two different util functions (this one + the one that returns the facetId without the prefix).